banners/wpde_desktop/C24_WPDE_Desktop_01/styles/styles.scss

```diff
@@ -0,0 +1,48 @@
+// This is the file where we import the theme-specific component styles
+@use '../../../../src/themes/Treedip/swatches/skin_default' with (
+	$slider: true,
+	$progress-bar: true,
+	$select-group: true,
+	$select-group-bubble: true,
+	$select-group-image-label: true,
+	$upgrade-to-yearly: true,
+	$fallback-banner: true,
+);
+@use '@src/themes/Treedip/variables/breakpoints';
+@use '@src/components/BannerConductor/banner-transition';
+@use 'Banner';
+@use 'MainBanner' with (
+	$banner-height: 394px,
+	$form-width: 300px
+);
+@use '../../../../src/themes/UseOfFunds/swatches/skin_default' as uof-default;
+@use '@src/themes/UseOfFunds/UseOfFunds';
+@use '@src/themes/Treedip/defaults';
+@use '@src/themes/Treedip/ButtonClose/ButtonClose';
+@use '@src/themes/Treedip/ProgressBar/ProgressBar' with (
```

Comment:
On line 58 of `ProgressBar.scss` there's a line height rule (`line-height: 26px;`) that you can delete. It's breaking the alignment of the inner text.